### ISSUE TITLE
ability to print sparkleformation template to file

### ISSUE THREAD

**Issue by markus7811** [envato/stack_master#226]
Hi,

we started to use stack_master and like it a lot! One thing would be great: the possibility to write the template down to json.

Because of the fantastic ability to specify SparklePacks within the stackmaster.yml we don't create the .sfn files so that we are not able to print to file with sfn print.

We need the json files, because we want to be able to lint the cloudformation before we validate or apply them

**Comment by patrobinson** [comment]
@markus7811 I hope @rbayerl has answered your question but I have another question.

We were planning on removing the ability to specify SparklePacks inside the `stack_master.yml` file because the "blessed" method to do this is like so:

```
require 'my_sparkle_pack'
pack = ::SparkleFormation::SparklePack.new(:name => 'my_sparkle_pack')
SparkleFormation.new(:my_stack, sparkle: pack) do
...
end
```

This didn't previously work due to a bug that was [fixed](https://github.com/sparkleformation/sparkle_formation/pull/241) in version 3.0.30 of SparkleFormation so I intended on reverting that change. So if you have another use case for it please let me know.

**Comment by markus7811** [comment]
Ok, how can i add several sparkle packs?

**Comment by stevehodgkiss** [comment]
@patrobinson I couldn't get sparkle packs working without using compiler options yesterday. I think some project specific dynamics were conflicting with the configured sparkle pack (configured like your example above in the template file). Adding code to the template to configure sparkle packs is tedious and really difficult to debug given the execution context in SparkleFormation, so I'd much prefer to keep compiler options for sparkle packs around.

**Comment by patrobinson** [comment]
Fixed in #245
